### ISSUE TITLE
Spots and Self-Spotting for SOTA / also via SMS-Text message

### ISSUE THREAD

**Issue by DocViper666** [ham2k/app-polo#42]
Please add the feature as already requested in #32 to show SOTA Spots (or better only Spots in selected activity) and allow self-spotting via the SOTA Spot system. 

I have seen that first code parts
`/ham2k/app-polo/blob/main/src/extensions/activities/sota/SOTAPostSpot.js`
are created but I looks like it is not implemented yet. It would require a SOTA SSO Username and Password fields in the settings. 

In the same course I would like to highlight the existance of **SMS Spot Gateways** that are operated by several users (like myself):
Documentation on the syntax is here.
https://www.sota.org.uk/Joining-In/SOTA-SMS-Format

Basically create from the current Callsign, Summit reference, Frequency and mode a SMS and send it to a selected phone number. Pre-requirement for the user is to have their phone number registered with the SOTA management team (MM0FMF Andy)

So one would need to option to select whether to send the spot via data services or via SMS.

73 Joe, OE5JFE

**Comment by kwirk** [comment]
So the feature of self-spotting via internet is in the latest beta for those signed up for testing in Android Play Store and iOS App Store, the 24.8.* version. Hopefully released more widely soon. Testers always welcome 😁 
![Screenshot_20240801-104321](https://github.com/user-attachments/assets/c094a1fe-c16d-472d-ae7d-c7b21ec9da2d)

**Comment by DocViper666** [comment]
Thanks for the heads up.
Just applied for beta. Logged in to SOTA SSO (I guess it took the username and password from the google account)?
Spot button is here. Very nice. Will give it a live test as soon as I have the chance.

If the spots will work too and S2S fields are populated from the spots this will be a great addition. And hurray 👍  for open source. 

Thanks for all the work!

**Comment by kwirk** [comment]
> If the spots will work too and S2S fields are populated from the spots this will be a great addition. And hurray 👍 for open source.


So only pulls other operators spots from POTA currently, but @sd is currently working on making it so each activity will be able to pull spots into a combined set. And plan is to have things like S2S, P2P, etc. populated automatically when picking the spot.



> I would like to highlight the existance of **SMS Spot Gateways**

So I was recently thinking it would be useful to be able to "share" your activity, and have details in text form which you could share in any chat app. For example WhatsApp is used for WWBOTA. Could use that similarly for sharing text to SMS app then for the SOTA spot gateways. Would need to think how this would work with people using different formats.

**Comment by sd** [comment]
The latest 24.8.0-pre version has SOTA spots
